common/shlibs

```diff
@@ -4146,3 +4146,4 @@ librz_search.so.0.3.4 rizin-0.3.4_1
 librz_bin.so.0.3.4 rizin-0.3.4_1
 libaravis-0.8.so.0 libaravis-0.8.21_1
 libLimeSuite.so.20.10-1 LimeSuite-20.10.0_1
+libwireplumber-0.4.so.0 wireplumber-0.4.9_1
```

Comment:
I don't think this is required, we usually only add entries here when a package needs them.